### PR TITLE
Update setup script

### PR DESCRIPTION
**Pull request by tjsilver**
## What's changed?
Following the recent [node upgrade](#1354), this PR:
- updates the setup script to include the V1 setup which was previously lost from the script
- updates the README to distinguish between V1 and V2 client-side tests

### General
- [ ] 🤖 Relevant tests added
- [ ] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

### Client
- [ ] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [ ] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ You can find the client for the Fronts tool in [fronts-client](/fronts-client).
 ### Setup (need to be done once)
 
 1. Install [NVM](https://github.com/creationix/nvm).
-1. Get credentials from [Janus](https://janus.gutools.co.uk/multi-credentials?&permissionIds=cmsFronts-dev,capi-api-gateway,frontend-dev) (`cmsFronts`, `capi` & `frontend`).
-1. Grant [code](https://permissions.code.dev-gutools.co.uk/) permissions (used for local builds as well).  You need:
+2. Get credentials from [Janus](https://janus.gutools.co.uk/multi-credentials?&permissionIds=cmsFronts-dev,capi-api-gateway,frontend-dev) (`cmsFronts`, `capi` & `frontend`).
+3. Grant [code](https://permissions.code.dev-gutools.co.uk/) permissions (used for local builds as well).  You need:
```
Comment (jonathonherbert) [inline README.md:13]: 😅

File: README.md [hunk 2]
```diff
@@ -103,6 +107,11 @@ In CODE the breaking news tool sends notifications to the "debug version" of the
 
 Breaking News is represented by a front called `breaking-news` which is considered to be a special case. It has a `Send Alert` button rather than a `Launch` button. Only one thing can be added to a collection at a given time. You cannot send the same alert twice, and snap links cannot be added / alerted on. Different collections represent different audience groups (eg by location or by subscription to different topics.) To add a new one, just create a container. New breaking news containers need to have the layout `breaking-news/not-for-other-fronts`.
 
+#### Running the app
+1. Run `npm install`
```
Comment (jonathonherbert) [inline README.md:111]: Do we need to take these install steps this if our setup script has already run an install for us?

**Review by jonathonherbert — APPROVED** [review]
Looks comprehensive! One minor comment 👍

File: README.md (hunk 2 above)
Comment (tjsilver) [reply README.md:111]: No, you're right, will take them out

**Pull request merged**